src/main/java/stepsdesign/PipelineSteps.java

```diff
@@ -659,4 +663,58 @@ public void clickOnCreateButton () {
   public void verifyInvalidConnectionErrorMessage(String errorMessageLocation) {
     CdfConnectionActions.verifyConnectionErrorMessage(errorMessageLocation);
   }
+
+  @Given("Open DataFusion Project with replication to configure pipeline")
+  public void openDataFusionProjectWithReplicationToConfigurePipeline() throws IOException, InterruptedException {
+    openCdf();
+    SeleniumDriver.getDriver().get(SeleniumDriver.getDriver().getCurrentUrl().replace(
+    SeleniumHelper.readParameters(ConstantsUtil.CDFURL), SeleniumHelper.readParameters(ConstantsUtil.REPLICATION_URL)));
+  }
+
+  @When("Enter input plugin property: {string} with pipelineName")
+  public void enterInputPluginPropertyWithPipelineName(String pluginProperty) {
+    WebElement pluginPropertyInput = PluginPropertyUtils.getInputPluginPropertyElement(pluginProperty);
+    String pipelineName = "TestPipeline-" + RandomStringUtils.randomAlphanumeric(10);
+    ElementHelper.sendKeys(pluginPropertyInput, pipelineName);
```

Comment:
In this Class we should only Define the steps, Create a method in ```CdfPluginPropertiesActions``` for the operation you want to perform.